### PR TITLE
chemical mixture type

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: instances/chemicalMixtureType/colloid.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/chemicalMixtureType/colloid",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/ChemicalMixtureType",
+    "definition": "A heterogeneous mixture in which one substance consisting of microscopically dispersed insoluble particles is suspended throughout another substance.",
+    "description": "A colloid is a mixture where very small particles of one substance are evenly distributed throughout another substance. They appear very similar to solutions, but the particles are suspended in the solution rather than fully dissolved. The difference between a     colloid and a suspension is that the particles will not settle to the bottom over a period of     time, they will stay suspended or float.",
```
Comment (apdavison) [inline instances/chemicalMixtureType/colloid.jsonld]: some cases of multiple       spaces between words here should be      fixed

File: instances/chemicalMixtureType/colloid.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "colloid",
+    "preferredOntologyIdentifier": null,
+    "synonym": null
+}
```
Comment (apdavison) [inline instances/chemicalMixtureType/colloid.jsonld]: need newline at the end of all files to make them valid JSON

**Review by apdavison — CHANGES_REQUESTED** [review]

File: instances/chemicalMixtureType/colloid.jsonld (hunk 1 above)
Comment (apdavison) [reply instances/chemicalMixtureType/colloid.jsonld]: `a . . . . colloid`, `of . . . . time`

File: instances/chemicalMixtureType/colloid.jsonld (hunk 2 above)
Comment (Peyman-N) [reply instances/chemicalMixtureType/colloid.jsonld]: My VS code keeps deleting them, would add it with github editor.

**Review by apdavison — APPROVED** [review]

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]
The instances and the schema look fine to me, but you are missing the corresponding instance that should be added to the terminology folder, here: https://github.com/HumanBrainProject/openMINDS_controlledTerms/tree/v1/instances/terminology.

**Review by lzehl — APPROVED** [review]
@Peyman-N and @UlrikeS91 I've made some changes to the definitions and descriptions. Please double check the changes and if you are okay with them let me know. From my side this PR is good to be merged then.

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**